packages/server/src/fhir/repo.ts

```diff
@@ -601,9 +602,10 @@ export class Repository extends FhirRepository<PoolClient> implements Disposable
     if (project) {
       resultMeta.project = project;
     }
-    const account = await this.getAccount(existing, updated, create);
-    if (account) {
-      resultMeta.account = account;
+    const accounts = await this.getAccounts(existing, updated, create);
+    if (accounts) {
+      resultMeta.account = accounts[0];
```

Comment:
🤔 

This is interesting.  I'm ok with it, but I also wonder if `accounts.length > 1` then we know they're using the new feature, and we could avoid setting legacy `account`?

Minor idle thought, definitely non-blocking